### ISSUE TITLE
CSS pattern %;-ms-text-size-adjust:100%; is falsely recognized as header variable

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#49]
The following CSS style:

```
style="width:100%;table-layout:fixed;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;"`
```

is falsely recognized as a SmartTemplate⁴ variable and removed. This will lead to faulty CSS

**Comment by RealRaven2000** [comment]
Below is a test version with a quick fix. I have improved the pattern matching, which only guarded against strings starting with numerals, such as `%1A%`, but not special characters like `%;-something%`. I have now restricted to only matching "headers" starting with letters, using the following regular expression:

```
msg = msg.replace(/%([a-zA-Z][\w-:=.]*)(\([^%]+\))*%/gm, replaceReservedWords);
```

[smartTemplate-fx-2.8.1pre40.zip](https://github.com/RealRaven2000/SmartTemplate4/files/4168137/smartTemplate-fx-2.8.1pre40.zip)

to install, unpack the contained xpi file and drag into Add-ons Manager.
